card/lib/card/spec_helper.rb

```diff
@@ -76,4 +76,33 @@ def render_card_with_args view, card_args={}, format_args={}, view_args={}
   def users
     SharedData::USERS.sort
   end
+
+  def in_phase opts, &event_block
+    $rspec_binding = binding
+    Card.class_eval do
+      alias_method :original_method_missing, :method_missing
+      def method_missing(m, *args)
```

Comment:
Method definitions must not be nested. Use `lambda` instead.<br>Use def without parentheses.